src/Files.App.Server/Files.App.Server.csproj

```diff
@@ -32,9 +37,16 @@
 
     <ItemGroup>
         <Manifest Include="app.manifest" />
-        <TrimmerRootAssembly Include="Files.App.Server"></TrimmerRootAssembly>
+        <TrimmerRootAssembly Include="Files.App.Server" />
         <PackageReference Include="Microsoft.Windows.CsWinRT" Version="2.0.4" />
         <PackageReference Include="Microsoft.Windows.CsWin32" Version="0.1.647-beta" PrivateAssets="all" />
     </ItemGroup>
 
+    <ItemGroup>
+        <None Include="..\Files.App.Server\bin\$(Platform)\$(Configuration)\$(TargetFramework)\win-$(Platform)\Files.App.Server.winmd">
```

Comment:
A workaround for appx toolchains. Without this the implementation of the server (Files.App.Server.dll) will be removed from the package layout.